[candi] Add chattr to prevent cleanup on Containerdv2

haosb opened the pull request:
## Description

Disable immutable flag on erofs files in cleanup node stage.

<!---
  Describe your changes in detail.

  Please let users know if your feature influences critical cluster components
  (restarts of ingress-controllers, control-plane, Prometheus, etc).
-->

## Why do we need it, and what problem does it solve?
<!---
  This is the most important paragraph.
  You must describe the main goal of your feature.

  If it fixes an issue, place a link to the issue here.

  If it fixes an obvious bug, please tell users about the impact and effect of the problem.
-->

Fix cleanup nodes with containerdV2.

## Why do we need it in the patch release (if we do)?

<!---
Describe why the changes need to be backported into the patch release.

If it doesn't matter whether the changes will be backported into the patch release, specify "Not necessarily".

Delete the section if the PR is for release, and not for the patch release.
-->

Cleanup containerdv2 nodes was broken.

## Checklist
- [ ] The code is covered by unit tests.
- [x] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [x] Changes were tested in the Kubernetes cluster manually.

## Changelog entries
<!---
  Describe the changes so they will be included in a release changelog.

  Find examples and documentation below, or visit the [Guidelines for working with PRs](https://github.com/deckhouse/deckhouse/wiki/Guidelines-for-working-with-PRs).
-->

```changes
section: candi
type: fix
summary: Disable immutable flag on erofs files in cleanup node stage.
impact_level: default
```

<!---
`impact_level: default` adds to changelog as usual, this is the default that can be omitted
`impact_level: high`    something important for users, the impact will be copied to "Know Before Update" section
`impact_level: low`     omitted in changelog YAML; note there is `type:chore` for chores

Tip for the section field:

  - <kebab-case of a module>, e.g. "cloud-provider-aws", "node-manager"
  - "ci", has forced low impact
  - "docs", includes website changes, should have low impact
  - "candi"
  - "deckhouse-controller"
  - "dhctl"
  - "global-hooks"
  - "go_lib"
  - "helm_lib"
  - "jq_lib"
  - "shell_lib"
  - "testing", has forced low impact
  - "tools", has forced low impact

Find changed sections:

gh pr diff   $PULL_REQUEST_NUMBER   |
  egrep "^([+]{3} b|[-]{3} a)/" |
  cut -d/ -f2- |
  sed 's#^ee/##' |
  sed 's#^fe/##' |
  sed 's#^modules/##' |
  sed 's#[0-9][0-9][0-9]-##' |
  egrep -v 'Makefile' |       # add file exclusion here
  cut -d/ -f1 |
  sort |
  uniq

Find all possible sections (excluding ci):

node -e 'console.log(require("./.github/scripts/js/changelog-find-sections.js")().join("\n"))'
-->

deckhouse-BOaTswain commented:
Aye, aye, @haosb. I've started the workflow for label 'e2e/run/vsphere'...
<!-- workflow_start -->

deckhouse-BOaTswain commented:
Aye, aye, @haosb. I've started the workflow for label 'e2e/run/yandex-cloud'...
<!-- workflow_start -->

deckhouse-BOaTswain commented:
Aye, aye, @haosb. I've started the workflow for label 'e2e/run/vsphere'...
<!-- workflow_start -->

haosb commented:
Before:
```
rm: cannot remove '/var/lib/containerd/io.containerd.snapshotter.v1.erofs/snapshots/79/fs/lib64/libinih.so.0': Read-only file system
rm: cannot remove '/var/lib/containerd/io.containerd.snapshotter.v1.erofs/snapshots/79/fs/lib64/libm.so.6': Read-only file system
rm: cannot remove '/var/lib/containerd/io.containerd.snapshotter.v1.erofs/snapshots/79/fs/lib64/libmount.so.1': Read-only file system
rm: cannot remove '/var/lib/containerd/io.containerd.snapshotter.v1.erofs/snapshots/79/fs/lib64/libmount.so.1.1.0': Read-only file system
rm: cannot remove '/var/lib/containerd/io.containerd.snapshotter.v1.erofs/snapshots/79/fs/lib64/libparted.so.2': Read-only file system
rm: cannot remove '/var/lib/containerd/io.containerd.snapshotter.v1.erofs/snapshots/79/fs/lib64/libparted.so.2.0.5': Read-only file system
rm: cannot remove '/var/lib/containerd/io.containerd.snapshotter.v1.erofs/snapshots/79/fs/lib64/libpcre2-8.so.0': Read-only file system
rm: cannot remove '/var/lib/containerd/io.containerd.snapshotter.v1.erofs/snapshots/79/fs/lib64/libpcre2-8.so.0.14.0': Read-only file system
rm: cannot remove '/var/lib/containerd/io.containerd.snapshotter.v1.erofs/snapshots/79/fs/lib64/libreadline.so.8': Read-only file system
rm: cannot remove '/var/lib/containerd/io.containerd.snapshotter.v1.erofs/snapshots/79/fs/lib64/libreadline.so.8.2': Read-only file system
```

After:
```
[root@17828400149-1-con-1-30-master-0 redos]# bash /var/lib/bashible/cleanup_static_node.sh --yes-i-am-sane-and-i-understand-what-i-am-doing
Removed "/etc/systemd/system/multi-user.target.wants/bashible.timer".
Removed "/etc/systemd/system/multi-user.target.wants/d8-shutdown-inhibitor.service".
Removed "/etc/systemd/system/multi-user.target.wants/sysctl-tuner.timer".
Failed to disable unit: Unit file old-csi-mount-cleaner.service does not exist.
Failed to disable unit: Unit file d8-containerd-cgroup-migration.service does not exist.
Removed "/etc/systemd/system/multi-user.target.wants/containerd-deckhouse.service".
Removed "/etc/systemd/system/multi-user.target.wants/kubelet.service".
Failed to stop old-csi-mount-cleaner.service: Unit old-csi-mount-cleaner.service not loaded.
Failed to stop old-csi-mount-cleaner.timer: Unit old-csi-mount-cleaner.timer not loaded.
Failed to stop d8-containerd-cgroup-migration.service: Unit d8-containerd-cgroup-migration.service not loaded.

Broadcast message from root@localhost on pts/1 (Thu 2025-09-18 15:29:16 MSK):

The system is going down for reboot at Thu 2025-09-18 15:30:16 MSK!

Reboot scheduled for Thu 2025-09-18 15:30:16 MSK, use 'shutdown -c' to cancel.
```

deckhouse-BOaTswain commented:
Cherry pick PR [15531](https://github.com/deckhouse/deckhouse/pull/15531) to the branch [release-1.71](https://github.com/deckhouse/deckhouse/tree/release-1.71) successful!

deckhouse-BOaTswain commented:
Cherry pick PR [15532](https://github.com/deckhouse/deckhouse/pull/15532) to the branch [release-1.72](https://github.com/deckhouse/deckhouse/tree/release-1.72) successful!
